src/Installer/Database/PostgresDatabase.php

```diff
@@ -22,10 +22,11 @@ class PostgresDatabase extends AbstractDatabase
 
     public function getCreateDatabaseCommand()
     {
+        $pass     = !empty($this->pass) ? 'PGPASSWORD='.escapeshellarg($this->pass).' ' : '';
```

Comment:
I've seen both, and without is more common in my recent experience.
That said, without is more depended on your shell supporting it (which most do).